src/index.html

```diff
@@ -1,13 +0,0 @@
-<!DOCTYPE html>
```

Comment:
This file is not used. The actual file that is read by the browser is in `index.html`